scipy/special/_logsumexp.py

```diff
@@ -1,5 +1,8 @@
 import numpy as np
 from scipy._lib._util import _asarray_validated
+from scipy._lib._array_api import (array_namespace, size as xp_size, is_jax,
```

Comment:
```suggestion
from scipy._lib._array_api import (array_namespace, size as xp_size,
```
WIll commit with `[skip ci]` if the rest passes.